<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dfsButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>700</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>深度优先搜索</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="oradea">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Oradea</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="zerind">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ZerInd</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="arad">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Arad</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="timisoara">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Timisoara</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lugoj">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Lugoj</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mehadia">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Mehadia</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dobreta">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Dobreta</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sibiu">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Sibiu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fagaras">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Fagaras</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rimnicu">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rimnicu Vicea</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="craiova">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Craiova</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pitesti">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitesti</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bucharest">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bucharest</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="giurgiu">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>590</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Giurgiu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="urziceni">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Urziceni</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="neamt">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Neamt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lasi">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Lasi</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="vaslui">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>268</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vaslui</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hirsova">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>420</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Hirsova</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="eforie">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>550</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Eforie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bfsButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>700</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>广度优先搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>700</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="greedyButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>700</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>贪婪算法</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
